tests/e2e/downloadExcel.spec.ts

```diff
@@ -22,6 +22,6 @@ test("can download Excel file for scenario results", async ({ page, baseURL }) =
   await download.path();
 
   // Filename is derived from parameter values, in this case the defaults
-  const expectedFilename = "daedalus_United Kingdom_sars_cov_1_none_none_26200.xlsx";
+  const expectedFilename = "daedalus_Thailand_sars_cov_1_none_none_22000.xlsx";
```

Comment:
Could make this a pattern rather than fixed string, which allows the default hospital capacity to vary if data ever gets updated. Though I don't think that's going to be an issue for a while!